www/README.md

```diff
@@ -11,14 +11,14 @@ Source for [harp.gl](https://heremaps.github.io/harp.gl) "landing" page
 
 ## Production architecture
 
-Main part of "app" is hosted on [Heremaps org Github Pages](https://heremaps.github.io/harp.gl).
+Main part of "app" is hosted on [Heremaps org Github Pages](https://www.harp.gl).
 
 Examples and docs are hosted on S3, behind Cloudfront.
 
 (Looks like) Cloudfront is the frontend for `harp.gl`, works as proxy for:
 
 * `https://www.harp.gl/docs/*` - rev-proxied to S3
-* rest - rev-proxied to `https://heremaps.github.io/harp.gl/`
+* rest - rev-proxied to `https://www.harp.gl`
```

Comment:
This is not correct. The rest is now also proxied to s3, just to a different folder?!

Reply:
Sorry, didn't read all the text first time, just replaced a link. Now this README should reflect how it really works after change.